### ISSUE TITLE
Remove extraneous dependencies

### ISSUE THREAD

**Issue by bcdarwin** [neuropoly/spinalcordtoolbox#2680]
Of the packages listed in `requirements.txt`, `xlrd`, `xlwt`, and `xlutils` are semi-deprecated in favour of `openpyxl` except for compatibility with very old Excel formats.  Of these, only `xlutils` appears in the codebase and only in one place, so at least the other two likely could be removed from the installation.

**Comment by Drulex** [comment]
I am in favor with the idea of migrating the 3 deprecated/in-process-of-deprecation packages mentioned above (see https://groups.google.com/forum/#!msg/python-excel/P6TjJgFVjMI/g8d0eWxTBQAJ) in favor of `openpyxl`. I took a quick look at the code and noticed the following:

1. None of them are actually used in the code base (@bcdarwin had mentioned that `xlutils` was used somewhere, is this still the case? I don't see it). In that case we don't event need to add `openpyxl` as a dependency. Note that `pandas.ExcelWriter()` does use the `xlwt` engine to write to an excel file, but that is not a direct dependency of this project.
2. I looked at the other requirements and noticed that `cryptography` and `psutils` are also not used anywhere in the codebase. Do we want to change the scope of this issue and remove those as well?

Let me know if I am missing something, I just recently joined the project and trying to get familiarized by attacking simple issues such as this one.
